docs/source/release_notes/release-notes.rst

```diff
@@ -44,22 +44,40 @@ develop
     *    - Type
          - Change
 
-    *    - |userbreak|
-         - Timelock clients now report tritium metrics for the lock requests with the prefix ``LockService`` instead of ``RemoteLockService``.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2284>`__)
+    *    -
+         -
 
-    *    - |improved|
-         - LockServerOptions now provides a builder, which means constructing one should not require overriding methods.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2284>`__)
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
-    *    - |new|
+=======
+v0.56.0
+=======
+
+12 September 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    - |devbreak| |new|
          - TimelockServer now exposes the `LockService` instead of `RemoteLockService` if using the synchronous lock service.
-           This will provide a more comprehensive API required by the large internal product.
+           This will provide a more comprehensive API which is required by the large internal products.
+           Any product which extends a TransactionManager will have to make changes to their class definition in order to support this update.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2284>`__)
 
-    *    - |new|
-         - Oracle will now validate connections by running the test query when getting a new connection from the HikariPool.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2301>`__)
+    *    - |devbreak| |new| |deprecated|
```

Comment:
Discussed offline with @amarzoca, but might be good to split off deprecated and dev break out. Not sure what the devbreak is exaclty, @jnelson15 for SA.